### DISCUSSION TITLE
Multiple specs on a single page, separated into tabs in Docusaurus Plugin.

### DISCUSSION THREAD

**Discussion by Flepsz** [scalar/scalar#1795] [Ideas]
I think the possibility of multiple specifications on the same page is important. Imagine an API with microservices, adding each service to each page, in my opinion, is not the best way.

Here I have an idea:

![Idea Preview](https://github.com/scalar/scalar/assets/70305395/43d9ff3a-0342-4871-b2e5-a5d1305fff38)

**Comment by marclave** [comment]
@hanspagel @cameronrohani @amritk 

okay, so I like this idea... should we allow people to pass in an array of spec files? and we merge them all?

**Reply by amritk** [reply]
This could use the new sidebar me and Hans were talking about

**Reply by hanspagel** [reply]
Merging would cause a lot of headaches! We could add support for multiple specification, but that’s a bigger left. Basically touches every single file. 👀 I see the use case, though.

**Reply by hanspagel** [reply]
BTW here is the relevant issue: https://github.com/scalar/scalar/issues/1851
